src/components/shared/Title/index.jsx

```diff
@@ -1,7 +1,17 @@
 import style from "./index.module.scss";
 
-const Title = ({ text, isSecondary }) => (
-  <h1 className={isSecondary ? style.secondary : style.title}>{text}</h1>
+const Title = ({ text, isSecondary, isLowerCase }) => (
+  <h1
+    className={
+      isSecondary
+        ? style.secondary
+        : isLowerCase
+        ? style.lowerCase
+        : style.title
```

Comment:
avoid using nested ternary operators, you can use a helper library like I mentioned in another comment, to ease applying more complex conditions like this one